pkg/ovs/ovsctl/appctl_test.go

```diff
@@ -110,3 +125,134 @@ IPv6 ND Extension: No`,
 		})
 	}
 }
+
+func TestTrace(t *testing.T) {
+	assert := assert.New(t)
+	srcMAC, _ := net.ParseMAC("11:11:11:11:11:11")
+	dstMAC, _ := net.ParseMAC("11:11:11:11:11:22")
+	srcIP := net.ParseIP("1.1.1.2")
+	dstIP := net.ParseIP("2.2.2.2")
+	tc := []struct {
+		name          string
+		req           *TracingRequest
+		expectedFlow  string
+		expectedError string
+	}{
+		{
+			name: "Trace IP Packet-success",
+			req: &TracingRequest{
+				InPort:              "9",
+				SrcIP:               srcIP,
+				DstIP:               dstIP,
+				SrcMAC:              srcMAC,
+				DstMAC:              dstMAC,
+				Flow:                "in_port=3,tcp,tcp_dst=22,",
+				AllowOverrideInPort: true,
+			},
+			expectedFlow:  "dl_src=11:11:11:11:11:11,dl_dst=11:11:11:11:11:22,in_port=3,tcp,tcp_dst=22,,nw_ttl=64,nw_src=1.1.1.2,nw_dst=2.2.2.2,",
+			expectedError: "",
```

Comment:
When the `expectedError` is "", this line can be removed. So does to `expectedFlow`